bottles/backend/managers/installer.py

```diff
@@ -249,13 +249,18 @@ def __step_run_script(config: BottleConfig, step: dict):
                 return False
 
         logging.info(f"Executing installer script…")
-        subprocess.Popen(
+        result = subprocess.run(
             f"bash -c '{script}'",
             shell=True,
             cwd=ManagerUtils.get_bottle_path(config),
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-        ).communicate()
+            capture_output=True,
+        )
+        logging.info(f"Output: \n{result.stdout.decode()}\n")
+        if result.returncode != 0:
```

Comment:
change this to `return result.returncode == 0` this will return `True` if the condition meet and `False` otherwise. If you look at how the `install` function executes the installer' steps, it checks if one of them returns `False` https://github.com/bottlesdevs/Bottles/blob/2bd8b60608661228ea3e804edbd7ab0dc05faa0c/bottles/backend/managers/installer.py#L421 and then return a proper `Result` object according to that. 

Currently, there is no easy way to show a message to the user graphically. It could be possible by making each step returning a tuple (boolean, message) and have the `install` function append that message to the Result object, then updating the installer view in the frontend to use that message if available https://github.com/bottlesdevs/Bottles/blob/2bd8b60608661228ea3e804edbd7ab0dc05faa0c/bottles/frontend/windows/installer.py#L173

Reply:
adding the line `return result.returncode == 0` results in a known failure (exit 1) in the script not even putting a warning in the Console, and just continuing with the rest of the manifest steps. 

Same goes for adding
`return False`
or
```
      return Result( 
         False, data={"message": "Installer is not well configured."} 
     ) 
```
Bottles just continues on with the rest of the manifest regardless of the failure


It looks like there is something missing to handle the return at this point, as it continues regardless of any state returned